index.html

```diff
@@ -2,13 +2,13 @@
 <html lang="es">
   <head>
     <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./css/styles.css">
     <title>Mi primera página web</title>
   </head>
   <body>
-    <h1>Hola, mundo!</h1>
-    <p>Soy Kevin y esta es mi primera página web en HTML.</p>
+    <h1 id="main__title--index" class="main__title">Hola, mundo!</h1>
```

Comment:
```suggestion
    <h1 id="main__title__index" class="main__title">Hola, mundo!</h1>
```
Trataría de mantener la consistencia al nombre clases. Usar siempre `__` o siempre `--`